BuildROS2/build_ros2_base.sh

```diff
@@ -32,7 +32,9 @@ export CXX="/usr/bin/clang++-13"
 export MY_LINKER_FLAGS="-latomic "\
 "-Wl,-rpath=\${ORIGIN} "\
 "-Wl,-rpath-link=/usr/lib/x86_64-linux-gnu "\
-"-Wl,-rpath-link=/usr/lib "
+"-Wl,-rpath-link=/usr/lib "\
+"-Wl,-rpath-link=/lib32 "\
+"-Wl,-rpath-link=/lib/i386-linux-gnu " 
```

Comment:
I don't think its necessary to support 32-bit builds. UE5 does not support 32 bit at all, and UE4 editor is 64-bit only, leaving a fairly dead edge-case where someone would try to run Unreal on a decades old machine.